Apps/GatewayApi/src/Controllers/PhsaController.cs

```diff
@@ -58,9 +59,9 @@ public PhsaController(IDependentService dependentService)
         [HttpGet]
         [Authorize(Policy = UserProfilePolicy.Read)]
         [Route("dependents/{hdid}")]
-        public ActionResult<IEnumerable<DependentModel>> GetAll(string hdid)
+        public ActionResult<RequestResult<IEnumerable<DependentModel>>> GetAll(string hdid)
```

Comment:
NIT: I'm not sure why we insist using RequestResult, I thought we are trying to move away from it